commands/action/nom.js

```diff
@@ -22,64 +22,24 @@ module.exports = class NomCommand extends Command {
         const disgust =
             disgustP[Math.round(Math.random() * (disgustP.length - 1))];
         const recipient = message.content.split(/\s+/g).slice(1).join(' ');
+        const embed = new Discord.MessageEmbed();
 
-        if (args.length < 1) {
-            const embed = new Discord.MessageEmbed()
-                .setColor('#FBCFCF')
-                .setImage(disgust);
+        if (
+            args.length < 1 ||
+            message.mentions.users.first() == message.author
+        ) {
+            embed.setColor('#FBCFCF');
+            embed.setImage(disgust);
             return message.channel.send(
                 `${message.author} noms on... themselves..?`,
                 { embed: embed }
             );
-        } else if (message.mentions.users.first() == message.author) {
-            const embed2 = new Discord.MessageEmbed()
-                .setColor('#FBCFCF')
-                .setImage(disgust);
-            return message.channel.send(
-                `${message.author} noms on... themselves..?`,
-                { embed: embed2 }
-            );
         } else if (message.mentions.users.first() == this.client.user) {
-            await axios
```

Comment:
Why remove axios? how does it pull images now?

Reply:
No need to use axios when Snekfetch is being used.